src/components/Checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,66 @@
+import styled from '@emotion/styled';
+import checkmark from './checkmark.svg';
+import Label from '../Label';
+
+const StyledCheckbox = styled.input<CheckboxProps>`
+  -moz-appearance: none;
+  -webkit-appearance: none;
+  -o-appearance: none;
+
+  ${({ theme, error }) => {
+    return {
+      display: 'flex',
+      alignItems: 'center',
+      justifyContent: 'center',
+      height: '22px',
+      width: '22px',
+      border: `2px solid ${theme.semantic.color.border.default}`,
+      borderRadius: theme.semantic.border.radius.small,
+
+      ...(error && {
+        border: `2px solid ${theme.semantic.color.border.error}`,
+      }),
```

Comment:
This statement works, but I would turn it into tertiary statement for clarity
```suggestion
      ...(!!error ? {
        border: `2px solid ${theme.semantic.color.border.error}`,
      } : undefined),
```